.circleci/config.yml

```diff
@@ -195,9 +181,8 @@ jobs:
             - packages/*/dist
             - packages/*/build
             - packages/*/lib
-            - packages/*/node_modules
             - packages/client-api-schema/types
-            - node_modules
+            - .yarn
```

Comment:
Thinking about this some more, I think the right thing to do is to completely remove circle caching `save_cache` and `restore_cache`, since the dependencies are pulled down in the `checkout_code` step.